frontend/public/components/app.jsx

```diff
@@ -150,7 +148,6 @@ class App extends React.PureComponent {
 
   render() {
     const { isNavOpen } = this.state;
-    let DefaultPage = isKubevirt() ? KubevirtDefaultPage : DefaultPage; // eslint-disable-line
```

Comment:
I removed redirect to virtual machines page and we will land on tectonic default page. To keep the VM as default page we would have to find out accessible namespaces/projects which would be a bigger change.